org2jekyll.el

```diff
@@ -198,22 +208,22 @@ The `'%s`' will be replaced respectively by the blog entry name, the author, the
 (defalias 'org2jekyll/create-draft! 'org2jekyll-create-draft)
 
 (defun org2jekyll--list-dir (dir)
```

Comment:
As I see it, supposedly "org2jekyll--list-dir" should do more than "find-file", leave it as it is now to serve as a reminder to extend it later.

Further "listing" is valid extension point: instead of a plain dired buffer, a separate one which list meta info about drafts/posts would be much more useful.